themes/cooperkid/cooperkid.theme.bash

```diff
@@ -27,11 +29,13 @@ function git_short_sha() {
 
 function prompt() {
 	local return_status=""
-	local ruby="${red}$(ruby_version_prompt)${reset_color}"
-	local user_host="${green}\h @ \w${reset_color}"
-	local git_branch="$(git_short_sha)${cyan}$(scm_prompt_info)${reset_color}"
+	local ruby
+	ruby="${red}$(ruby_version_prompt)${reset_color}"
```

Comment:
just to stick to the format, how about adding the question marks to this line as well?

Reply:
Sounds good! :+1: